.github/workflows/rust.yml

```diff
@@ -6,65 +6,30 @@ on:
   pull_request:
     branches: [ master ]
 
-jobs:
-  check:
```

Comment:
While not as important as `fmt` or `clippy` jobs it is sometimes still great to have a fast `check` job for faster turnaround times with the CI.